hack/golint.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+CMD=${0##*/}
+
+# TODO(efried): Accept optional targets override
+targets=$(go list -f '{{ .Dir }}' ./...)
```

Comment:
I see where this is going off.. the go list here isn't seeing the `unexport GOFLAGS` in the makefile.

This should work, or unexporting GOFLAGS
```suggestion
targets=.
```

Reply:
Unexporting `GOFLAGS` fixes this an the other thing that was holding me back. Thanks.